tests/cloner_test.go

```diff
@@ -30,9 +36,77 @@ var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:compo
 	f := framework.NewFrameworkOrDie(namespacePrefix)
 
 	var sourcePvc *v1.PersistentVolumeClaim
+	var targetPvc *v1.PersistentVolumeClaim
+	AfterEach(func() {
+		if sourcePvc != nil {
+			By("[AfterEach] Clean up target PVC")
+			err := f.DeletePVC(sourcePvc)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetPvc != nil {
+			By("[AfterEach] Clean up target Block PVC")
```

Comment:
Oops forgot to change the message to not include block here, should be Clean up target PVC